index.html

```diff
@@ -676,6 +676,11 @@ <h2>Manual Commands</h2>
             the [@username]'s direct messages. The embed contains the command
             author's name, the [reason] if provided, and a shiny gold star!
           </li>
+	  <li>
+	    `sus` - This command sends a message embed to the channel the usersaid 
+            the command in. The command will say that a certain color looks sus just 
+            like in Among Us Emergency Meetings.
+	  </li>
```

Comment:
```suggestion
          <li>
            `sus` - This command sends a message embed to the channel the
            user called the command in. The command will say that a certain color
            looks sus just like in Among Us Emergency Meetings.
          </li>
```
Indentation, verbiage.